integreat_cms/cms/models/abstract_tree_node.py

```diff
@@ -282,7 +287,23 @@ def move(self, target, pos=None):
         # Moving a node can modify all other nodes via raw sql queries (which are not recognized by cachalot),
         # so we have to invalidate the whole model manually.
         invalidate_model(self.__class__)
-        super().move(target=target, pos=pos)
+        try:
+            with db_mutex(self.__class__.__name__):
+                super().move(target, pos)
+        except DBMutexError as e:
+            logger.error("Could not retrieve database mutex to save %r", self)
```

Comment:
```suggestion
```
Ah, since the exception is logged in the page actions anyway, maybe we don't need to log it here again?